CHANGES.md

```diff
@@ -44,6 +44,7 @@ Core Grammars:
 New Grammars:
 
 - added 3rd party TTCN-3 grammar to SUPPORTED_LANGUAGES [Osmocom][]
+- added 3rd party Odin grammar to SUPPORTED_LANGUAGES [clsource](https://github.com/clsource)
```

Comment:
Please use the `[]` linking style, like all the other attributions.  You'll add your url below.

Reply:
Done 👍